### DISCUSSION TITLE
webhelp-responsive/tasks/carMaintenance

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#6] [Q&A]
# webhelp-responsive/tasks/carMaintenance

Avoid costly repairs by constantly performing simple maintenance tasks.

http://127.0.0.1:5500/webhelp-responsive/tasks/carMaintenance.html

**Comment by syeda-git** [comment]
Test comment 11/20
